framework/auth/oauth_scopes.py

```diff
@@ -212,8 +214,8 @@ class ComposedScopes(object):
                      CoreScopes.NODE_PREPRINTS_WRITE, CoreScopes.PREPRINT_REQUESTS_WRITE, CoreScopes.WIKI_BASE_WRITE)
 
     # Preprints collection
-    PREPRINT_METADATA_READ = (CoreScopes.PREPRINTS_READ, CoreScopes.PREPRINT_CITATIONS_READ, CoreScopes.IDENTIFIERS_READ,)
-    PREPRINT_METADATA_WRITE = PREPRINT_METADATA_READ + (CoreScopes.PREPRINTS_WRITE, CoreScopes.PREPRINT_CITATIONS_WRITE,)
+    PREPRINT_METADATA_READ = (CoreScopes.PREPRINTS_READ, CoreScopes.PREPRINT_CITATIONS_READ, CoreScopes.IDENTIFIERS_READ, CoreScopes.METRICS_READ,)
```

Comment:
Similar point to above, is there a better place to put these? Adding its own composed scope seemed overkill...

Reply:
If we were using scopes to handle permissions rather than system tags, then we would put these into their own composed scope. But, as things are, this seems like a perfectly good place for them.